src/alfasim_sdk/_internal/alfasim_sdk_utils.py

```diff
@@ -92,3 +93,15 @@ def get_extras_default_required_version() -> str:
     Default is greater or equal to current alfasim-sdk version.
     """
     return f">={get_current_version()}"
+
+
+def get_required_python_version() -> str:
+    """
+    Return the required python version.
+    """
+    major, minor, micro = (
+        sys.version_info.major,
+        sys.version_info.minor,
+        sys.version_info.micro,
+    )
+    return f">={major}.{minor}.{micro}"
```

Comment:
I don't think we need to include the micro version in the required version. Also, to be safe in case the plugin contains compiled code, better include an upper bound to the Python version:


```suggestion
def get_required_python_version() -> str:
    """
    Return the required python version to be included in plugins metadata when using
    the CLI to generate a template.
    """
    # Assume most plugins will use compiled code and include an upper bound.
    major, minor, *_ = sys.version_info
    return f">={major}.{minor},<{major}.{minor + 1}"
```